### PR TITLE
AT_113_Java | Entity > Export Destination: Create a new record and cancel it

### PR DESCRIPTION
**Pull request by LMalushkina**
Added class EntityExportDestinationCancelRecordTest
https://trello.com/c/PRItVMTa/473-at113java-entity-export-destination-create-a-new-record-and-cancel-it

### CODE REVIEW

File: src/test/java/EntityExportDestinationCancelRecordTest.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+import static utils.ProjectUtils.*;
+import static utils.TestUtils.scrollClick;
+
+public class EntityExportDestinationCancelRecordTest extends BaseTest {
+
+    @Test
+    public void testCancelRecord() {
+
+        start(getDriver());
```
Comment (Burdishka) [inline src/test/java/EntityExportDestinationCancelRecordTest.java:19]: now "start" method we don't need to use

File: src/test/java/EntityExportDestinationCancelRecordTest.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        findElement(By.id("int")).sendKeys("1");
+        findElement(By.id("decimal")).sendKeys("1.10");
+        findElement(By.id("date")).click();
+        findElement(By.id("date")).clear();
+        findElement(By.id("date")).sendKeys("09/06/2021");
+        findElement(By.id("datetime")).click();
+        findElement(By.id("datetime")).clear();
+        findElement(By.id("datetime")).sendKeys("09/06/2021 20:05:23");
+
+        WebElement cancelButton = getDriver().findElement(By.xpath("//button[contains(text(),'Cancel')]"));
```
Comment (Burdishka) [inline src/test/java/EntityExportDestinationCancelRecordTest.java:36]:
```suggestion
        WebElement cancelButton = findElement(By.xpath("//button[contains(text(),'Cancel')]"));
```

File: src/test/java/EntityExportDestinationCancelRecordTest.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        scrollClick(getDriver(), cancelButton);
```
Comment (Burdishka) [inline src/test/java/EntityExportDestinationCancelRecordTest.java:37]: So if you make static import of scrollClick() then delete TestUtils in 20 or add here TestUtils and then delete static import in 12

File: src/test/java/EntityExportDestinationCancelRecordTest.java [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+        List<WebElement> records = getDriver().findElements(By.xpath("//td[@class ='pa-list-table-th']"));
+
+        Assert.assertEquals(records.size(), 0);
```
Comment (Burdishka) [inline src/test/java/EntityExportDestinationCancelRecordTest.java:41]: I check your user story and you tell us "In page Export Destination shouldn't be record with entered values. Entered record shouldn't be in the Recycle Bin". So as I see you need to add one more verification in the Recycle Bin.

**Review by Burdishka — COMMENTED** [review]
Don't forget to refresh your branch before push updates to PR

File: src/test/java/EntityExportDestinationCancelRecordTest.java (hunk 4 above)
Comment (LMalushkina) [reply src/test/java/EntityExportDestinationCancelRecordTest.java:41]: Ок, спасибо

File: src/test/java/EntityExportDestinationCancelRecordTest.java (hunk 1 above)
Comment (LMalushkina) [reply src/test/java/EntityExportDestinationCancelRecordTest.java]: Done

File: src/test/java/EntityExportDestinationCancelRecordTest.java (hunk 2 above)
Comment (LMalushkina) [reply src/test/java/EntityExportDestinationCancelRecordTest.java]: Done

File: src/test/java/EntityExportDestinationCancelRecordTest.java (hunk 4 above)
Comment (LMalushkina) [reply src/test/java/EntityExportDestinationCancelRecordTest.java:34]: Done

File: src/test/java/EntityExportDestinationCancelRecordTest.java (hunk 3 above)
Comment (LMalushkina) [reply src/test/java/EntityExportDestinationCancelRecordTest.java]: Done

**Review by PRoman-86 — APPROVED** [review]

**Review by Burdishka — APPROVED** [review]

**Pull request merged**